tests/phpunit/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilderTest.php

```diff
@@ -98,6 +98,10 @@ public function test_it_builds_and_dump_the_xml_configuration(): void
 
     public function test_it_preserves_white_spaces_and_formatting(): void
     {
+        if (PHP_EOL === "\r\n") {
```

Comment:
minor:

we already have:

```php
if (PHP_OS_FAMILY === 'Windows') {
        $this->markTestSkipped('Not interested in profiling on Windows');
}
```
and 

```php
if ('\\' === DIRECTORY_SEPARATOR) {
```

what do you think about using one of those methods to detect windows and not introduce the 3rd one?

Reply:
This is not to detect Windows, but to test specifically against `PHP_EOL`. Windows build under WSL might have proper line endings, for example. Makes sense?

Reply:
yep, thanks